stl/inc/format

```diff
@@ -122,10 +122,7 @@ constexpr const _CharT* _Parse_arg_id(const _CharT* _Begin, const _CharT* _End,
 
 template <class _CharT, _Parse_spec_callbacks<_CharT> _Callbacks_type>
 constexpr const _CharT* _Parse_align(const _CharT* _Begin, const _CharT* _End, _Callbacks_type&& _Callbacks) {
-    // done with parsing, or reached the end of a replacement field.
-    if (_Begin == _End || *_Begin == '}') {
-        return _Begin;
-    }
+    // Preconditions: _Begin != _End && *_Begin != '}'

```

Comment:
It's not a policy yet, but I've been trending toward using `_STL_INTERNAL_CHECK` to document checkable preconditions and get a bit of validation coverage since `_STL_INTERNAL_CHECK` is active while running the test suite. You may want to think about changing this later.

Reply:
it would also fix CI :D

Reply:
Also, oops: it looks like there are `_Parse_align` test cases that exercised this conditional that need to be removed.

Reply:
yep, just did that.